.github/workflows/beaker.yml

```diff
@@ -56,7 +56,7 @@ on:
         type: string
       acceptance_runs_on:
         description: the runner group used for acceptance jobs run on
-        default: ubuntu-20.04
+        default: ubuntu-22.04
```

Comment:
podman can't run unprivileged containers on 20.04

Reply:
Perhaps [puppet_metadata](https://github.com/voxpupuli/puppet_metadata) should generate the host it runs on and be aware that EL7 needs to run on 20.04 while running anything else on 22.04? Pretty sure you can set the runs-on part via a matrix.

Reply:
Or: We merge this into a v3 branch and introduce this to modules after they drop EL7 from CI?